environment.yml

```diff
@@ -5,6 +5,7 @@ dependencies:
   - python=3.12
   - jupyterlab
   - jupytext
+  - jupyterlab-myst
```

Comment:
I'd be hesitant to tweak our dependencies this late, but as far as I'm aware it will not break anything. Users missing this will just not see nice admonitions..? So okay in my book. :)

Reply:
Yeah it's just a visual thing -- the existing notebooks already use admonitions, so may as well IMO?

Reply:
Go for it. :)